consensus/debugging.go

```diff
@@ -194,7 +189,7 @@ func (m *consensusModule) sendGetMetadataStateSyncMessage(_ *messaging.DebugMess
 		}
 		valAddress := cryptoPocket.AddressFromString(val.GetAddress())
 		if err := m.stateSync.SendStateSyncMessage(stateSyncMetaDataReqMessage, valAddress, requestHeight); err != nil {
-			m.logger.Error().Err(err).Str("proto_type", "GetMetaDataRequest").Msg("failed to send StateSyncMessage")
+			m.logger.Error().Err(err).Str("proto_type", "StateSyncMetadataRequest").Msg("failed to send StateSyncMessage")
```

Comment:
This isn't the actual proto type: `StateSyncMetadataRequest`.

Reply:
but it is: https://github.com/pokt-network/pocket/blob/main/consensus/types/proto/state_sync.proto

Reply:
Good point